state/metrics.go

```diff
@@ -66,4 +63,7 @@ type Metrics struct {
 	// The duration of accesses to the state store labeled by which method
 	// was called on the store.
 	StoreAccessDurationSeconds metrics.Histogram `metrics_bucketsizes:"0.0002, 10, 5" metrics_buckettype:"exp" metrics_labels:"method"`
+
+	// The duration of event firing related to a new block
+	BlockEventsDurationSeconds metrics.Gauge
```

Comment:
Can we come with a better name for it?

Like `FireBlockEventsDelay` or something like that?

Moreover, all metrics should be in seconds by default. See the metric you have removed. The time unit should be part of the description, not the metric name.

Reply:
I added the `Seconds` suffix because it seems to be the norm in most of our existing metrics.

The one I've removed seems to be an exception in many ways (not having a units suffix, using `UnixNano, and being measured in milliseconds. Indeed I wasted several hours because of that mess. So, next time I work with a metric I want do know in which units it is measured.

Having the units in the description is OK, but as stated, most of our metrics have a suffix with the units, so we should be consistent.

Reply:
Following your proposal, I'll rename it `BlockEventsDurationSeconds` to `FireBlockEventsDelaySeconds`